index.html

```diff
@@ -14,58 +14,41 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css">
     <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
     <link href='https://pydelhi.org/assests/css/main.css' rel='stylesheet'>
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
```

Comment:
Why did we need `jQuery`? Can't we make do with vanilla JavaScript?

Reply:
working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Reply:
> working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Reply:
> Why did we need `jQuery`? Can't we make do with vanilla JavaScript?

working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Reply:
We can use the browsers' `fetch` API instead. Decreases dependencies.

Reply:
as discussed earlier, using of JQuery is fine